test/check-machines-nics

```diff
@@ -139,6 +140,27 @@ class TestMachinesNICs(VirtualMachinesCase):
         b.wait_not_present(".pf-c-modal-box")
         b.wait_not_present("#vm-subVmTest1-network-1-mac")
 
+        # Detach NIC when the VM is shutoff, Also check this issue: https://bugzilla.redhat.com/show_bug.cgi?id=1791543
+        self.performAction("subVmTest1", "off")
+        # Attach since no NIC now
+        m.execute("virsh attach-interface --domain subVmTest1 --type network --source default --model virtio --mac 52:54:00:4b:73:5f --config")
```

Comment:
--config and --current should be the same for offline Vms. So you can remove it without changing the functionality.

Reply:
OK, but in last test run, --current brought a behavior which is the VM disappear after refreshing. I could not reproduce it manually, and when automation, it seems not to be reproduced every time. So I try to use --config instead of --current, and the cases all passed in five times, so I guess maybe we meet a issue in libvirt, which will happen when attaching nic with --current multi-times, and not 100% reproducible. I am not sure, so I think we should run test to verify that. So what's your opinion?